docs/content/patterns/selectable-table-with-batch-actions/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Selectable tables with batch actions
-description: Selectable tables allow users to select one or more rows simultaneously and perform batch actions against the selected rows.
+description: Selectable tables allow users to select one or more rows simultaneously and perform batch actions against the selected rows. A Batch action is any action that can be performed to every row of a table.
```

Comment:
Nitpicking on semantics but "any action that can be performed to every row of a table." is not the same as "against every selectable row" as mentioned below. One suggests all rows in the table, the other suggests all with a checkbox

Reply:
"Batch" should be lowercase too

Reply:
Agreed

Reply:
```suggestion
description: Selectable tables allow users to select one or more rows simultaneously and perform batch actions against the selected rows. A batch action is any action that can be performed against any selectable row of a table.
```